### PR TITLE
Get rid of application.js

### PR DESCRIPTION
**Pull request by jmhooper**
The only thing that app.js did was included the service provider JS on every page and include the utils.js which is now gone. This commit only includes the service provider js where necessary and removes application.js.

### CODE REVIEW

**Review by zachmargolis — APPROVED** [review]
huge! LGTM

File: app/javascript/packs/service-provider-form.js [hunk 1]
```diff
@@ -132,6 +132,10 @@ function logoUploadSetup() {
   const logoInput = document.querySelector('.logo-input-file');
   const logoPreview = document.querySelector('.input-preview');
 
+  if (!logoInput) {
```
Comment (aduth) [inline app/javascript/packs/service-provider-form.js:135]: Can we drop the condition on line 168 of this file, which was added in a previous pull request to support this script being loaded on pages other than the form (no longer necessary)?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -174,8 +174,5 @@
         </div>
       </div>
     </footer>
-
-    <%= javascript_include_tag 'js/main.js' %>
```
Comment (aduth) [inline app/views/layouts/application.html.erb:178]: Did this previously not exist? 🤔

**Review by aduth — APPROVED** [review]
👍

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (jmhooper) [reply app/views/layouts/application.html.erb:178]: yeah, this was asking for something that does not exist

Comment (jmhooper) [reply app/views/layouts/application.html.erb:178]: actually, double checking, this may be USWDS JS. Let me make sure I haven't broken everything

Comment (aduth) [reply app/views/layouts/application.html.erb:178]: > `identity-style-guide` is in the asset path and this is pulling from there.

Ah, tricky!

**Pull request merged**